automation/artifacts.repo

```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: 2022 Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Don't add license in automation - delete this directory first.

Reply:
https://github.com/oVirt/vdsm/pull/308